app/app.go

```diff
@@ -500,6 +557,43 @@ func (app *AkashApp) registerUpgradeHandlers() {
 		// configure store loader that checks if version == upgradeHeight and applies store upgrades
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
+
+	// ica upgrade
+	const upgradeName = "akash_ica_upgrade"
```

Comment:
are upgrades ordered by name?  if so, can you make this `"01-ica-upgrade"` to leave room for `"02-foo"`?

Reply:
No, it doesn't care about the order. It just cares about the upgrade height. But we can have a better upgrade name.